build/compiler.js

```diff
@@ -296,12 +297,8 @@ function getConfig({target, env, dir, watch, cover}) {
                         {pragma},
                       ],
                       cover && require.resolve('babel-plugin-istanbul'),
-                      target === 'node' &&
+                      target === 'web' &&
```

Comment:
This should run in the browser compilation now. Instead of directly instrumenting the server bundle, it passes data to the server compilation out-of-band.